packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -350,6 +379,14 @@ const ReleaseDetailsBody = () => {
                   <Td>
                     <Typography>{entry.contentType.displayName || ''}</Typography>
                   </Td>
+                  <Td>
+                    <Flex>
```

Comment:
I think this Flex probably belongs directly in the ReleaseActionOptions component. That's my bad. Could you update it in the CMReleasesContainer as well?